deployments/mdpresentation-viewer.yml

```
@@ -0,0 +1,14 @@
+---
+services:
+  ocis:
+    depends_on:
+      mdpresentation-viewer:
+        condition: service_completed_successfully
+
+  mdpresentation-viewer:
+    image: jankaritech/mdpresentation-viewer
```

```suggestion
    image: jankaritech/mdpresentation-viewer:2.0.0
```

Although it should work without a tag, I think it's recommended to add the tag of the specific image, or create a "latest" tag on Docker hub if you don't want to update this script when there are other releases in the future